src/index.ts

```diff
@@ -197,7 +197,7 @@ function attr(input: Element): Node[] {
 
   return attrs.map((attr): Node => ({
     name: '[' + cssesc(attr.name, {isIdentifier: true}) + '="' + cssesc(attr.value) + '"]',
-    penalty: 1
+    penalty: 0.5
```

Comment:
Why 0.5?

Reply:
So that it gets prefered to classes but not ids.

Reply:
Okay, I think attributes may be more important when classes. Maybe create another PR with config option for it?